switchapi/CMakeLists.txt

```diff
@@ -68,8 +68,8 @@ target_include_directories(switchapi_o PRIVATE
     ${SDE_INSTALL_DIR}/include/target-sys
 )
 
-find_library(TDI tdi ${SDE_INSTALL_DIR}/lib)
```

Comment:
I think it's good to provide HINTS options with the path where these libraries can be found. Referring to the documentation here - https://cmake.org/cmake/help/latest/command/find_library.html

Also, should we specify REQUIRED option for tdi libraries since there is no point in proceeding without TDI libraries
REQUIRED
New in version 3.18.

Stop processing with an error message if nothing is found, otherwise the search will be attempted again the next time find_library is invoked with the same variable.